cmd/peek-submission/main.go

```diff
@@ -1,4 +1,4 @@
-package main
+package peek
```

Comment:
I don't think this will even work.
Have you tried to actually run the CMD like this?